### ISSUE TITLE
Merge in various wG projects people have started

### ISSUE THREAD

**Issue by scottwalters** [AlliumCepa/webgui#3]
These projects could get their own tickets if started, but I wanted to list some of them:

https://github.com/frodwith/dbix-class-webgui
https://github.com/haarg/webgui-assets
https://github.com/frodwith/helpdesk-rewrite

I'll add others here as I track them down.

**Comment by scrottie** [comment]
Looks like Imager is in.  Image::Magick is still used in lib/WebGUI/Image.pm and sbin/thumbnailer.pl and needs to be replaced there (Imager work isn't complete).

**Comment by lindleyw** [comment]
With Debian 8 and fresh new install of WebGUI from GitHub, cpanm complains of no version information for Image::Magick, and aborts installation.

The problem seems to be that, for whatever reason, something is missing.  After manually installing several ImageMagick packages, along with libperl-dev (so -lperl works in the linker), and then custom symlinking the configuration header file, I was able to successfully install the Image::Magick module.

```
sudo apt-get install libperl-dev libmagickcore-6-arch-config
sudo apt-get install libgraphics-magick-perl libmagickcore-dev
sudo ln -s /usr/include/x86_64-linux-gnu/ImageMagick-6/magick/magick-baseconfig.h /usr/include/ImageMagick-6/magick/magick-baseconfig.h 
sudo cpanm Image::Magick  # success!
```

Your baseconfig.h might be in a different place for the ln -s command, if so, try:

```
sudo updatedb
locate magick-baseconfig.h
```

**Comment by scrottie** [comment]
Nope, Image::Magick is coming out.  Too many problems over too long of period of time.  The code doesn't use it any more, but I need to talk to Colin Kuskie about how to handle including that in Task::WebGUI.  7 still needs it.  Maybe there needs to be a Task::WebGUI8.  I saw the automated failure report.

**Comment by lindleyw** [comment]
Roger that. All the messy manual stuff was sufficient to let me run through the rest of the installer correctly, though.  It's good that everything else worked flawlessly: a big improvement over a few months ago! Salud.
